Workbooks/Resource Groups/Sql databases/Sql databases.workbook

```diff
@@ -216,48 +202,41 @@
               }
             },
             {
-              "columnMatch": "microsoft.sql/servers/databases-Basic-storage_percent Timeline",
+              "columnMatch": "microsoft.sql/servers/databases-Basic-physical_data_read_percent Timeline",
               "formatter": 5,
               "formatOptions": {
                 "showIcon": true
               }
             },
             {
-              "columnMatch": "microsoft.sql/servers/databases-Basic-physical_data_read_percent|microsoft.sql/servers/databases-Basic-log_write_percent",
+              "columnMatch": "microsoft.sql/servers/databases-Basic-connection_failed",
               "formatter": 8,
               "formatOptions": {
                 "min": 0,
-                "palette": "orange",
+                "palette": "red",
                 "showIcon": true
               },
               "numberFormat": {
-                "unit": 1,
+                "unit": 0,
                 "options": {
                   "style": "decimal",
                   "maximumFractionDigits": 2
                 }
               }
             },
             {
-              "columnMatch": "microsoft.sql/servers/databases-Basic-physical_data_read_percent Timeline",
-              "formatter": 5,
-              "formatOptions": {
-                "showIcon": true
-              }
-            },
-            {
-              "columnMatch": "microsoft.sql/servers/databases-Basic-log_write_percent Timeline",
+              "columnMatch": "microsoft.sql/servers/databases-Basic-connection_failed Timeline",
               "formatter": 5,
               "formatOptions": {
                 "showIcon": true
               }
             }
           ],
+          "rowLimit": 10000,
```

Comment:
do we need this line?

Reply:
probably good to have a limit (even though it's very large)